src/components/PurchaseOrder/PO.js

```diff
@@ -124,6 +131,26 @@ class PO extends Component {
     return false;
   }
 
+  closeOrder = (values) => {
```

Comment:
`values` is a bad name, doesn't reflect what's the values. This is callback, please be more specific. Or maybe it's better to use Destructuring assignment here: `{reason, note}`